decidim-proposals/config/locales/en.yml

```diff
@@ -26,7 +27,8 @@ en:
       proposal_answer:
         answer: Answer
       proposals_copy:
-        copy_proposals: I understand that this will import all proposals from the selected component to the current one and that this action can't be reversed.
+        copy_proposals: I understand that this will import all proposals from the
```

Comment:
There are a lot of unrelated changes, can you check them please?

Reply:
Hi @mrcasals I've just run i18n-tasks... and it made a lot of changes in all modules. I finally undid all changes in other modules but as I added text into decidim-proposals had to keep the result of i18n-tasks.
This was the exact command `bundle exec i18n-tasks normalize --locales en` so the result should be correct no?

Reply:
Hi @mrcasals any news here? (cc/ @oriolgual )

Reply:
Where did you run the command from? You need to run it from the root of the project, not inside `decidim-proposals` 😄

Reply:
Hold on, I see these changes if I run the command from the root folder on `master`, let me check it.

Reply:
@tramuntanal check this PR! https://github.com/decidim/decidim/pull/4975 Sorry for the inconvenience!

Reply:
OK, thanks a lot for investigating :+1: 
I'll have to wait for #4975 to be re-added correct?

Reply:
@tramuntanal correct! PR is already merged, thanks for the review!